airflow/utils/dag_cycle_tester.py

```diff
@@ -25,6 +25,17 @@
 
 
 def test_cycle(dag):
+    """
+    A wrapper function of check_cycle for backward compatibility reason.
+    New code should use check_cycle instead, since this function name test_cycle starts with test_ and pytest
```

Comment:
```suggestion
    A wrapper function of `check_cycle` for backward compatibility reason.
    New code should use `check_cycle` instead, since this function name `test_cycle` starts with 'test_' and pytest
```